docs/rst/user_guide/data/intro.rst

```diff
@@ -23,37 +21,42 @@ data.
 Accessing shipped data
 ----------------------
 
-Eradiate ships with a series of data sets located in its ``resources/data``
-directory. Some of the larger data sets consist of aggregates of many NetCDF
-files, while others are stand-alone NetCDF files. In order to provide a simple
-and unified interface, Eradiate references data sets in a registry which can be
-queried using the :mod:`~eradiate.data` module.
+Eradiate ships with a series of data sets managed its global data store.
+
+.. code-block:: python
 
-Data sets are grouped by category, *e.g.* solar irradiance spectrum, absorption
-spectrum, etc. The complete list of registered categories can be found in the
-reference documentation for the :mod:`~eradiate.data` module. Each data set is
-then referenced by an identifier unique in its category. The pair
-(category, identifier) therefore identifies a data set completely.
+   from eradiate.data import data_store
 
-To open a specific data set, the :func:`eradiate.data.open` function should be
-used:
+This global data store aggregates multiple subordinated data stores based on
+the size and maturity level of data files they manage.
+List a data store' registered data sets by reading its ``registry`` property,
+i.e.:
 
 .. code-block:: python
 
-   import eradiate
-   ds = eradiate.data.open("solar_irradiance_spectrum", "thuillier_2003")
+   list(data_store.stores["small_files"].registry)
 
-The :func:`~eradiate.data.open` function can also be used to load user-defined
-data at a known location:
+for small data files and:
 
 .. code-block:: python
 
-   ds = eradiate.data.open("path/to/my/data.nc")
+   list(data_store.stores["large_files_stable"].registry)
```

Comment:
Quite inconvenient IMO: this is typically something which should be implemented as a feature of the `eradiate data info` command.

Reply:
I agree

Reply:
I created eradiate/eradiate-issues#149 to address that